### ISSUE TITLE
issue with stylesheets.

### ISSUE THREAD

**Issue by ColdHam** [mmuehlberger/tutsplus-getting-started-with-react-native#1]
Went through tutorial had one issue with stylesheets, seems I am missing something?
In the index.io.js you require the styles from the stylesheet.js using
var styles = require('./style');

and within the styles.js you export default
export default {
  global: require('./styles/global'),
  listView: require('./styles/list-view'),
  detailView: require('./styles/detail-view')
}

and then you access the styles within the index.io.js by 
styles.global.mainContainer for example. 
This however does not work for me. I ended up having to access via
styles.default.global.default.mainContainer

I logged the styles variable after requiring it, and the export default seems to be wrapping the exported object in a default : {} object property. Why is this not the case with your code?

![screen shot 2016-01-25 at 9 03 40 am](https://cloud.githubusercontent.com/assets/1812922/12552996/e91bbeb8-c342-11e5-893c-78940ae4f69a.png)
![screen shot 2016-01-25 at 9 05 56 am](https://cloud.githubusercontent.com/assets/1812922/12552995/e914e908-c342-11e5-845d-7ed527473262.png)

**Comment by mmuehlberger** [comment]
I have looked into it and found that React Native upgraded to Babel 6 in Release 0.16 (I'm using 0.15).

The way I used it is technically incorrect, although done by many, because I was combining ES6 exporting syntax with CommonJS require syntax.

There is actually an [article](https://medium.com/@kentcdodds/misunderstanding-es6-modules-upgrading-babel-tears-and-a-solution-ad2d5ab93ce0#.9z353057p) about this exact problem, with 3 proposed solutions:

1. Do it as you did, by appending `.default` to the require
2. Use 100% ES6 modules with `import styles from './styles'` syntax
3. Use 100% CommonJS syntax with `module.exports` instead of `exports default`
